### DISCUSSION TITLE
VSCode command: Open slide # from file y

### DISCUSSION THREAD

**Discussion by svrooij** [marp-team/marp#548] [Ideas]
I’ve been playing around with using Marp in combination with a live coding demo, where I script like everything in VSCode.
It would be cool if you guys could implement some sort of live server that allows opening the presentation on a specific slide from a VSCode command.

`Open slide {filename} {slideNumber}`

**Comment by yhatt** [comment]
Currently we have no plans to execute live server from VS Code extension because it may become too complex for managing server.

Marp team is focused on "creating presentations", and does not prioritize "doing presentations" including live demos. If necessary, please consider implementing such features with third-party tools. (https://github.com/marp-team/awesome-marp#community)

**Reply by svrooij** [reply]
I was hoping for the marp cli supported as a command, but well. There are other options available.
